lib/metrics/metric_api.rb

```diff
@@ -60,8 +60,9 @@ def create(definition)
       # Query metric definitions by tags
       # @param tags [Hash]
       # @return [Array[MetricDefinition]]
-      def query(tags)
-        @client.http_get("/metrics/?type=#{@type}&tags=#{tags_param(tags)}").map do |g|
+      def query(tags = nil)
+        tags_filter = tags.nil? ? '' : "&tags=#{tags_param(tags)}"
+        @client.http_get("/metrics/?type=#{@type}#{tags_filter}").map do |g|
```

Comment:
Ok, my bad did not recognize `#{}` as such, as I was too much focused of the # in html-links to anchors.